learn-new.md

```diff
@@ -0,0 +1,259 @@
+		<h3 id="observing-ballerina-programs"><a href="/learn/observing-ballerina-programs/observing-your-application-with-prometheus-grafana-and-jaeger/">Observing Ballerina Programs
+</a></h3>
+		<p>Basics of the observability functionalities that are provided for Ballerina programs.</p>
+		<h3 id="running-ballerina-programs-in-the-cloud"><a href="/learn/running-ballerina-programs-in-the-cloud/code-to-cloud/">Running Ballerina Programs in the Cloud
+</a></h3>
+		<p>The cloud offerings for running Ballerina programs.</p>
+		<h3 id="managing-dependencies"><a href="/learn/managing-dependencies/">Managing Dependencies </a></h3>
+		<p>Details of declaring and managing dependencies and using the local repository.</p>
+		<h3 id="publishing-packages-to-ballerina-central"><a href="/learn/publishing-packages-to-ballerina-central/">Publishing Packages to Ballerina Central</a></h3>
+		<p>Details of publishing your library package to Ballerina Central.</p>
+		<h3 id="calling-java-code-from-ballerina-and-vice-versa"><a href="/learn/calling-java-code-from-ballerina-and-vice-versa/">Calling Java Code from Ballerina and Vice Versa</a></h3>
+		<p>Instructions on the supported interoperability features.</p>
+
+</div>
+
+</div>
+</div>
+
+<div id="Tokyo" class="tabcontent">
+ <div class="container">
+<div class="row balraw" style="background-color:aliceblue;padding: 25px 25px 15px 25px;margin-bottom:30px;width:97.04%">
+
+		<!--<h3 id="language-guide"><a href="/learn/language-concepts/">Language Guide</a></h3>
+		<p>An elaborate textual guide to the Ballerina language.</p>-->
+		<h3 id="reference-guide-by-examples"><a href="/learn/by-example/">Reference by Examples</a></h3>
+		<p>A series of examples that serve as a reference guide for language constructs, concepts, and standard library modules.</p>
+		<h3 id="library-documentation"><a href="https://lib.ballerina.io/">Library Documentation</a></h3>
+		<p>Ballerina library API documentation.</p>
+		<h3 id="the-bal-tool"><a href="/learn/cli-documentation/cli-commands/">CLI Documentation</a></h3>
+		<p>Details of all the CLI commands of the <code class="highlighter-rouge language-plaintext">bal</code> tool.</p>
+		<h3 id="specifications"><a href="/learn/platform-specifications/">Platform Specifications</a></h3>
+		<p>Details of the Ballerina language specifications and proposals.</p>
+        <h3 id="style-guide"><a href="/learn/style-guide/coding-conventions/">Style Guide</a></h3>
+		<p>Best practices to follow when formatting Ballerina code.</p>
+		<h3 id="language-introduction-slides"><a href="/learn/language-introduction/">Language Introduction</a></h3>
+		<p>Presentation slides on the Ballerina language that you can use to talk about the language.</p>
+        <!--<h3 id="blogs-and-articles"><a href="https://blog.ballerina.io/">Blogs/Articles</a></h3>
+		<p>Provides details of all the CLI commands of the `bal` tool.</p>-->
+
+</div>
+
+</div>
+</div>
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Shall we remove above unecessary new lines?

Reply:
Yeah, I removed them, Thank you